src/components/Modal/Modal.tsx

```diff
@@ -72,7 +72,7 @@ export const BaseModal: ChakraComponent<
 
           <ModalFooter>
             <ButtonGroup>
-              <Button id="modal-close-btn" onClick={onClose}>
+              <Button id="modal-close-btn" height="40px" onClick={onClose}>
```

Comment:
Can you update this and the open button to `height="10"`? It's the same 40px value but in the design token name.

Reply:
Just a reminder that this update is now in the `react18-and-chakra` base branch so pull the latest updates instead of making this update.